src/features/snaps/components/UserCard.tsx

```diff
@@ -0,0 +1,67 @@
+import { Box, Button, Flex, Text } from '@chakra-ui/react';
```

Comment:
I have created similar component called - AccountCard, It will get merged soon, you can reuse it. May be you might need to make trustScore prop field non-mandatory